server/form-pages/assess/reviewApplication/reviewApplicationAndDocumentsPage.test.ts

```diff
@@ -14,7 +22,42 @@ describe('Review', () => {
     })
   })
 
-  itShouldHaveNextValue(new Review({}), '')
+  describe('initialize', () => {
+    beforeEach(() => {
+      assessment.application = overwriteApplicationDocuments(assessment.application, [])
+    })
+
+    it('should call the API to fetch documents for the application', async () => {
+      const getDocumentsMock = jest.fn().mockResolvedValue(documentFactory.buildList(3))
+      const applicationService = createMock<ApplicationService>({ getDocuments: getDocumentsMock })
+
+      await Review.initialize({}, assessment, token, { applicationService })
+
+      expect(applicationService.getDocuments).toHaveBeenCalledWith(token, assessment.application)
+    })
+
+    it('should add the full Document objects given the document filenames supplied on the application', async () => {
+      const fullSelectedDocumentObjects = [
+        documentFactory.build({ fileName: 'documentA' }),
+        documentFactory.build({ fileName: 'documentB' }),
+      ]
+      assessment.application = overwriteApplicationDocuments(assessment.application, [
+        { fileName: 'documentA' } as Document,
+        { fileName: 'documentB' } as Document,
+      ])
+
+      const getDocumentsMock = jest.fn().mockResolvedValue(fullSelectedDocumentObjects)
+
+      const applicationService = createMock<ApplicationService>({ getDocuments: getDocumentsMock })
+
+      const page = await Review.initialize({}, assessment, token, { applicationService })
+
+      // eslint-disable-next-line
```

Comment:
What rule are we disabling here? Can we just disable the specific rule, rather than everything?

Reply:
Yeah this slipped through, I'll add `no-explicit-any`